test/parallel/test-tls-min-max-version.js

```diff
@@ -13,6 +13,7 @@ const DEFAULT_MAX_VERSION = tls.DEFAULT_MAX_VERSION;
 
 function test(cmin, cmax, cprot, smin, smax, sprot, proto, cerr, serr) {
   assert(proto || cerr || serr, 'test missing any expectations');
+  const where = (new Error()).stack.split('\n')[2].replace(/[^(]*/, '');
```

Comment:
Could you add a comment with an example of what the contents of this variable might end up being? For example, `// e.g. (test/parallel/test-tls-min-max-version.js:1234)`?

Reply:
ok